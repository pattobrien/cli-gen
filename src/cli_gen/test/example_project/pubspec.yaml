name: example_project
description: A sample command-line application.
version: 1.0.0
publish_to: none

environment:
  sdk: ^3.4.0-132.0.dev

dependencies:
  cli_annotations:
    path: ../../../cli_annotations

dev_dependencies:
  lints: ^3.0.0
  test: ^1.24.0
