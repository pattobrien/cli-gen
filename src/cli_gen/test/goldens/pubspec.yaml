name: goldens
version: 1.0.0

environment:
  sdk: ^3.3.0

dependencies:
  cli_annotations: ^0.1.0-dev.4
  test: ^1.25.2

dev_dependencies:
  build: ^2.4.1
  build_runner: ^2.4.8
  build_test: ^2.2.2
  cli_gen: ^0.1.0-dev.4
  lints: ^3.0.0
  path: ^1.9.0
